br/br-log-architecture.md

```diff
@@ -108,18 +108,24 @@ Log backup generates the following types of files:
 .
 ├── v1
 │   ├── backupmeta
-│   │   ├── {min_restored_ts}-{uuid}.meta
-│   │   ├── {checkpoint}-{uuid}.meta
+│   │   ├── ...
+│   │   └── {resolved_ts}-{uuid}.meta
 │   ├── global_checkpoint
-│   │   ├── {store_id}.ts
-│   ├── {date}
-│   │   ├── {hour}
-│   │   │   ├── {store_id}
-│   │   │   │   ├── {min_ts}-{uuid}.log
-│   │   │   │   ├── {min_ts}-{uuid}.log
-├── v1_stream_truncate_safepoint.txt
+│   │   └── {store_id}.ts
+│   └── {date}
+│       └── {hour}
+│           └── {store_id}
+│               ├── ...
+│               └── {min_ts}-{uuid}.log
+└── v1_stream_truncate_safepoint.txt
 ```
 
+Explanation of the backup file directory structure:
+
+- `backupmeta`: contains backup metadata. The `resolved_ts` in the filename indicates the backup progress, meaning that data before this TSO has been fully backed up. However, note that this TSO only reflects the progress of certain shards.
+- `global_checkpoint`: represents the global backup progress. It records the latest point in time to which data can be restored using `br restore point`.
+- `{date}/{hour}`: stores backup data for the corresponding date and hour. When cleaning up storage, always use `br log truncate` instead of manually deleting data. This is because the metadata references the data in this directory, and manual deletion might lead to restore failures or data inconsistencies after restore.
```

Comment:
```suggestion
- `backupmeta`: stores backup metadata. The `resolved_ts` in the filename indicates the backup progress, meaning that data before this TSO has been fully backed up. However, note that this TSO only reflects the progress of certain shards.
- `global_checkpoint`: represents the global backup progress. It records the latest point in time to which data can be restored using `br restore point`.
- `{date}/{hour}`: stores backup data for the corresponding date and hour. When cleaning up storage, always use `br log truncate` instead of manually deleting data. This is because the metadata references the data in this directory, and manual deletion might lead to restore failures or data inconsistencies after restore.
```